### DISCUSSION TITLE
Does Whisper cpp WASM work on mobile?

### DISCUSSION THREAD

**Discussion by b1162** [ggml-org/whisper.cpp#2426] [Q&A]
Hi! We’re currently exploring Whisper cpp and we’re particularly interested in the WebAssembly (WASM) version. 
I am wondering if the WASM version should work on mobile devices. We’ve tried Whisper WASM using safari mobile, but it doesn’t seem to function properly on mobile. 

Could anyone clarify if the WASM build of Whisper cpp is expected to work on mobile, or if there are known issues or limitations in this environment?

**Comment by ggerganov** [comment]
Just tested on my iPhone and it appears to be working:

![005D3454-F977-45B0-927F-FB0982EC778B_1_102_o](https://github.com/user-attachments/assets/ce759bbb-f3b8-4c25-b113-df67c791955e)
